EZSwiftExtensionsTests/HashableTests.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  HashableTests.swift
+//  EZSwiftExtensions
+//
+//  Created by Rugen Heidbuchel on 01/08/2018.
+//  Copyright © 2018 Goktug Yilmaz. All rights reserved.
+//
+
+import XCTest
+@testable import EZSwiftExtensions
+
+class HashableTests: XCTestCase {
+    
+    func testHashString() {
```

Comment:
Lets also add a negative test case here (i.e. a situation in which XCTAssertNotEqual(s.hashString, String(s.hashValue)). 

But I cannot think of any such strings of the top of my head. Let me know if thats the case.

Reply:
I see. I still have to do the work to move it to swift 4.1 (its right now in swift 4.0 with XCode 9.2 as the build). 

I presume this file is a no op for prior XCode builds?